extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureUtils.java

```diff
@@ -40,11 +40,26 @@
 public class AzureUtils
 {
 
+  public static final String DEFAULT_AZURE_ENDPOINT_SUFFIX = "core.windows.net";
   @VisibleForTesting
   static final String AZURE_STORAGE_HOST_ADDRESS = "blob.core.windows.net";
+  public static final String DEFAULT_AZURE_BLOB_STORAGE_ENDPOINT_SUFFIX = "blob." + DEFAULT_AZURE_ENDPOINT_SUFFIX;
```

Comment:
how is this different from `AZURE_STORAGE_HOST_ADDRESS`? We can use that name itself instead of adding this one.